data/json/recipes/recipe_armor.json

```diff
@@ -127,6 +131,7 @@
   "reversible": true,
   "autolearn": true,
   "decomp_learn": 0,
+  "flags": ["BLIND_EASY"],
   "components": [
     [
       [ "modularvest", 1 ]
```

Comment:
the above recipes are just sliding plates in and out of pockets in the vest. seems very easy to do with your eyes closed.